dpnp/backend/kernels/dpnp_krnl_reduction.cpp

```diff
@@ -104,10 +140,28 @@ void func_map_init_reduction(func_map_t& fmap)
     fmap[DPNPFuncName::DPNP_FN_PROD][eft_FLT][eft_FLT] = {eft_FLT, (void*)dpnp_prod_c<float>};
     fmap[DPNPFuncName::DPNP_FN_PROD][eft_DBL][eft_DBL] = {eft_DBL, (void*)dpnp_prod_c<double>};
 
-    fmap[DPNPFuncName::DPNP_FN_SUM][eft_INT][eft_INT] = {eft_INT, (void*)dpnp_sum_c<int>};
-    fmap[DPNPFuncName::DPNP_FN_SUM][eft_LNG][eft_LNG] = {eft_LNG, (void*)dpnp_sum_c<long>};
-    fmap[DPNPFuncName::DPNP_FN_SUM][eft_FLT][eft_FLT] = {eft_FLT, (void*)dpnp_sum_c<float>};
-    fmap[DPNPFuncName::DPNP_FN_SUM][eft_DBL][eft_DBL] = {eft_DBL, (void*)dpnp_sum_c<double>};
+    // WARNING. The meaning of the fmap is changed. Second argument represents RESULT_TYPE for this function
+    // handle "out" and "type" parameters require user selection of return type
+    // TODO. required refactoring of fmap to some kernelSelector
+    fmap[DPNPFuncName::DPNP_FN_SUM][eft_INT][eft_INT] = {eft_LNG, (void*)dpnp_sum_c<int, int>};
```

Comment:
```suggestion
    fmap[DPNPFuncName::DPNP_FN_SUM][eft_INT][eft_INT] = {eft_INT, (void*)dpnp_sum_c<int, int>};
```

Reply:
@densmirn, I think, that it is okay, because if input.dtype == int32, return.dtype == int64 in numpy.

Reply:
I see, thanks.